enterprise/frontend/src/embedding-sdk/dev.md

```diff
@@ -0,0 +1,93 @@
+# Docs for development on the sdk
+
+These docs are for building the sdk locally, if you just want to use the sdk, please refer to the [sdk documentation](https://www.metabase.com/docs/latest/embedding/sdk/introduction).
+
+## Build
+
+You can build the sdk with `yarn build-embedding-sdk`.
+
+### Watch
+
+If you want to have it build when you change a file there are two options:
+
+#### build-embedding-sdk:watch
+
+`yarn build-embedding-sdk:watch` is the original command, the js output is fast, but the dts output is extremely slow and is not fixed by the dts fixup script on watch so typescript definitions may be broken.
+
+#### embedding-sdk:dev
+
+This is an _experimental_ command that should be much faster, it uses `tsc --incremental` to to generate the dts files and fixes them automatically by running the fixup script on watch.
+
+The `tsc` command will output a lot of errors, to keep the terminal output under control you may want to run the three different `embedding-sdk:dev:*` commands on different terminals.
+There is a VS code task named `Run embedding sdk dev commands` that does that
+
+## Storybook
+
+Storybook is a quick way of testing sdk components with hot reloading.
+
+Storybook expects an instance running on `localhost:3000` with some configuration:
+
+- on [/admin/settings/authentication/jwt](http://localhost:3000/admin/settings/authentication/jwt)
+  - Make sure "User Provisioning" is enabled.
+  - Set JWT secret to be "`0000000000000000000000000000000000000000000000000000000000000000`" in Admin > Authentication >
+    JWT > String used by the JWT signing key
+  - You will need to input a `JWT Identity Provider URI` even though it will not be used.
+  - Press "Save and enable"
+- on [/admin/settings/embedding-in-other-applications](http://localhost:3000/admin/settings/embedding-in-other-applications)
+  - Enable the sdk
+
+Then you can run `yarn storybook-embedding-sdk` to start storybook.
+
+Storybook will use the source files and not the built package.
+
+## E2E tests
+
+E2e tests for the sdk are currently done with cypress component tests.
+
+```bash
+yarn test-cypress-open-component-sdk
+```
```

Comment:
Optional: we could add the envs that are needed to run the EE tests to the docs:

```
MB_EDITION=ee
CYPRESS_ALL_FEATURES_TOKEN=
CYPRESS_NO_FEATURES_TOKEN=
```

Reply:
Good point! I always forget about it as I have them globally
b5c669d3072